projects.md

```diff
@@ -1,12 +1,90 @@
 ---
 layout: page
 title: Projects
-nav_order: 2
 permalink: /projects/
 main_nav: true
-secondary_nav: false
+nav_order: 4
+projects:
+  - name: Bitcoin Design Guide
+    description: The primary project we are working on, a resource for designers to create better Bitcoin products faster.
+    image: /assets/images/contribute/bitcoin-design-guide.svg
+    links:
+      - name: Site
+        link: https://bitcoin.design/guide
+      - name: Repository
+        link: https://github.com/BitcoinDesign/Guide
+  - name: Bitcoin Core
+    description: One of the oldest and best-maintained Bitcoin clients.
+    image: /assets/images/contribute/bitcoin-core.svg
+    links:
+      - name: Site
+        link: https://bitcoincore.org/
+      - name: Contribute
+        link: https://bitcoincore.org/en/contribute/
+  - name: BTCPay Server
+    description: BTCPay Server is proudly free and open-source, built and maintained by a world-wide community of passionate contributors.
```

Comment:
Just added a bit of context since the copy didn't tell what BTCPay Server actually is.
```suggestion
    description: BTCPay Server is a self-hosted, open-source Bitcoin payment processor, built and maintained by a world-wide community of passionate contributors.
```